alibi_detect/saving/loading.py

```diff
@@ -498,7 +505,7 @@ def resolve_config(cfg: dict, config_dir: Optional[Path]) -> dict:
         elif isinstance(src, dict):
             backend = cfg.get('backend', Framework.TENSORFLOW)
             if key[-1] in ('model', 'proj'):
-                obj = _load_model_config(src)
+                obj = _load_model_config(src, **kwargs)
```

Comment:
Just a general thought, but is it an issue if later we want multiple functions to receive `kwargs`? Is that likely to happen? I we had a `tf_load_Kwargs` would it be better? :thinking:

Reply:
Yeh true it would be an issue... but maybe we could just deal with that when/if we need to in the future?!